### PR TITLE
PDFConfiguration configs are passed explicitly to PDFManager.getPDFAs…

### PR DESCRIPTION
**Pull request by beetlecrunch**
…ByteArray

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.5.8
+sbt.version=1.6.2
```
Comment (galales) [inline project/build.properties]: I don't remember, which problems did we face last time with this upgrade?

Comment (TonioGela) [inline project/build.properties]: Was this intentional?

**Review by TonioGela — APPROVED** [review]

File: project/build.properties (hunk 1 above)
Comment (beetlecrunch) [reply project/build.properties]: This was upgraded during the test for CodeArtifact integration.
Just a typo, I fix it.

Comment (beetlecrunch) [reply project/build.properties]: [same](https://github.com/pagopa/interop-commons/pull/163#discussion_r913927175)

**Review by TonioGela — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**